openshift/ci-operator/build-image/cr.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "ClusterLogging"
+metadata:
+  name: "example"
```

Comment:
Do we need to explicitly define the namespace?

Reply:
apparently not - not sure if it is because i do `oc project openshift-logging` first, or use `oc -n openshift-logging create -f cr.yaml`

Reply:
both